backend/tests/vscode-events.spec.ts

```diff
@@ -81,8 +86,11 @@ const mockBasAPI: typeof bas = {
     getExtensionAPI: <T>(extensionId: string): Promise<T> => {
         return Promise.resolve(undefined);
     },
+    getAction: (id: string) => undefined,
+    getParameter: (key: string) => undefined,
+    performAction: <T = void>(action: BasAction, options?: { schedule?: boolean }) => undefined,
     actions: {
-        performAction: mockPerformAction,
+        performAction: <T = void>(action: BasAction, options?: { schedule?: boolean }) => undefined,
```

Comment:
instead of copying a complex signature may be able to reuse it.

`performAction: basAPI["performAction"]`